release-02/code/flag_tool.rb

```diff
@@ -0,0 +1,38 @@
+require 'sequel'
```

Comment:
I would add the shebang at this point: `#!/usr/bin/env ruby`. With it the script can be directly executed from the shell instead of calling `ruby flag_tool.rb`.